src/cyw43_config.h

```diff
@@ -115,3 +115,31 @@
 #ifndef CYW43_EVENT_POLL_HOOK
 #define CYW43_EVENT_POLL_HOOK
 #endif
+
+#if LWIP_DHCP
```

Comment:
I don't think this will work because lwip headers are not included until after this header files, so this define will not be set.

Probably best to just define `CYW43_DEFAULT_IP_ADDRESS` here as all other IP settings are defined, then in `cyw43_lwip.c` have something like:
```c
        #if LWIP_DHCP
        IP4_ADDR(&IP(ipconfig[0]), 0, 0, 0, 0);
        #else
        IP(ipconfig[0]).addr = PP_HTONL(CYW43_DEFAULT_IP_ADDRESS);
        #endif
```

Reply:
@peterharperuk I think this comment is still relevant, and my suggestion is still the way I think it should be done.

Reply:
Apologies, you're quite right. Should be fixed as suggested now